src/tests/tracing/eventpipe/processinfo/processinfo.cs

```diff
@@ -149,7 +149,9 @@ public static int Main()
                 // /path/to/corerun /path/to/processinfo.dll
                 // or
                 // "C:\path\to\CoreRun.exe" C:\path\to\processinfo.dll
-                string currentProcessCommandLine = $"{currentProcess.MainModule.FileName} {System.Reflection.Assembly.GetExecutingAssembly().Location}";
+                string currentProcessCommandLine = TestLibrary.Utilities.IsNativeAot
```

Comment:
It might be nicer to introduce IsSingleFile property (steal implementation from here: https://github.com/dotnet/runtime/blob/226e3ec0ad99ef97b91e8f72c681b9ea7398cb2e/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs#L205) and have this key off IsSingleFile. It could save work to someone. This logic is not native aot specific - it's single file specific.